tests/integration/replicator_test.py

```diff
@@ -228,9 +228,6 @@ def test_follow_replication_with_errors(self):
             ret = replicator.create_replication(
                 source_db=dbs,
                 target_db=dbt,
-                # Deliberately override these good params with bad params
```

Comment:
This removal doesn't seem correct. The test title is  since the test title suggests this is `test_follow_replication_with_errors` and this section says "override these good params with bad params" so I think these are supposed to be "broken" source and targets. Although the test is currently skipped anyway which probably means it doesn't matter much at all.

Reply:
@ricellis Right, although the test is currently skipped it would fail with this removal.  We have issue #114 for cleaning up the integration tests.  Maybe it's better to remove the whole test since it was using these optional parameters.

Reply:
Is there a change we can make that would mean if this test was re-enabled it would still fail? Can we specify a target that doesn't exist in object form?

Reply:
If there isn't I suggest we at the least add a "fail" here with a comment that the test needs a configuration change to do what it is supposed to in the event that anyone re-enables it.

Reply:
@ricellis Since the test is verifying that it will exit the follow loop on [these lines](https://github.com/cloudant/python-cloudant/blob/3535a6e35bc32f4cb2622a0295885e34ec4766e0/tests/integration/replicator_test.py#L234-L236), we can use an invalid target URL on line 204 i.e. `dbtarget = '_bad_db_name_'`